CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ All versions prior to 0.0.9 are untracked.
   `pyproject.toml` dependency source
   ([#300](https://github.com/trailofbits/pip-audit/pull/300))
 
+* Vulnerability fixing: the `--fix` flag now works for vulnerabilities found in
```

Comment:
This needs to be relocated under the "unreleased" section (this is my fault, due to the `2.3.3` release this morning).